### ISSUE TITLE
`Undefined index` when adding a file with embed link

### ISSUE THREAD

**Issue by mimamuh** [directus/api#1065]
# Bug Report
Adding a file via a link fails with `Undefined index`.

## Steps to Reproduce

1. Go to app `File Library`, then add a new file via `embed`
2. Add the link of the file (digital oceans spaces / aws s3)
3. See error

## Expected Behavior
Add the new file

## Actual Behavior
Api errors with:
```md
Undefined index: Prefix in /var/www/html/vendor/league/flysystem-aws-s3-v3/src/AwsS3Adapter.php on line 676
```
## Technical Details

- Directus API: 2.2.0
- Directus App: 7.6.0
- Web Server: nginx
- PHP Version: 7.2.18
- Database: MySQL 5.7
- Install Method: custom docker

**Comment by bjgajjar** [comment]
Not able to replicate in `API`. This issue will resolve once the `Docker` will be updated :)

**Comment by mimamuh** [comment]
Actually I tried it with a youtube link and also an image from a website and it works. So I guess it happened because of that:
1) I use a digital ocean space bucket
2) I uploaded a big file to the `do spaces bucket` to work around #999 
3) I copied the url and tried to add it to directus via the embed thing.

So the issue only happens when I use the same `bucket` as a source and destination. I didn't know it downloads the content from the link and upload it there. So I guess this issue can be closed as it is an edge-case which normally should never happen.
